### PR TITLE
fix(liveness): use updated Liveness oval algorithm

### PR DESCRIPTION
**Pull request by tjleing**
- [x] PR conforms to [Pull Request](https://github.com/aws-amplify/amplify-ui-android/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guidelines.

*Description of changes:*
use blazeface-reported ear positions to better calculate oval.  right now, works well at longer distances but messes up at short range (top is calculated incorrectly it seems?  shouldn't be negative).

*How did you test these changes?*
(Please add a line here how the changes were tested)

*Documentation update required?*
- [x] No
- [ ] Yes (Please include a PR link for the documentation update)

*General Checklist*
- [ ] Added Unit Tests
- [ ] Added Integration Tests
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tylerjroach** [comment]
I'm not sure what you were trying to do with the visibility annotation changes but they were unncecessary and actually brought internal code to be marked as public. I went ahead and pushed changes to revert all of those annotations.

Pushed a few other minor cleanups. Please take a look at my changes to confirm nothing unexpected was changed.

**Review by tylerjroach — APPROVED** [review]

File: liveness/src/test/java/com/amplifyframework/ui/liveness/ml/FaceDetectorTest.kt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        detector.generateBoundingBoxFromLandmarks(
+            faceBottom,
+            leftEye,
+            rightEye,
+            nose,
+            mouth,
+            leftEar,
+            rightEar
+        ).isEmpty
```
Comment (tylerjroach) [inline liveness/src/test/java/com/amplifyframework/ui/liveness/ml/FaceDetectorTest.kt:79]: Is this meant to be an assertion?

**Review by tylerjroach — CHANGES_REQUESTED** [review]

File: liveness/src/test/java/com/amplifyframework/ui/liveness/ml/FaceDetectorTest.kt (hunk 1 above)
Comment (tylerjroach) [reply liveness/src/test/java/com/amplifyframework/ui/liveness/ml/FaceDetectorTest.kt]: This is no longer relevant. I went ahead and fixed the test. Please confirm it looks as expected.

**Review by tylerjroach — APPROVED** [review]

File: liveness/src/main/java/com/amplifyframework/ui/liveness/ml/FaceDetector.kt [hunk 2]
```diff
@@ -405,7 +458,7 @@ internal class FaceDetector(private val livenessState: LivenessState) {
         private const val ANCHOR_OFFSET_Y = 0.5f
         private const val INPUT_SIZE_HEIGHT = 128
         private const val INPUT_SIZE_WIDTH = 128
-        private const val GOLDEN_RATIO = 1.618f
+        private const val GOLDEN_RATIO = 1.68f
```
Comment (ankpshah) [inline liveness/src/main/java/com/amplifyframework/ui/liveness/ml/FaceDetector.kt:461]: Golden Ratio should be 1.618

**Review by tylerjroach — APPROVED** [review]

File: liveness/src/main/java/com/amplifyframework/ui/liveness/ml/FaceDetector.kt (hunk 2 above)
Comment (tylerjroach) [reply liveness/src/main/java/com/amplifyframework/ui/liveness/ml/FaceDetector.kt]: reverted

**Review by tylerjroach — APPROVED** [review]

**Review by ankpshah — APPROVED** [review]

**Pull request merged**